lego/apps/lending/managers.py

```diff
@@ -0,0 +1,44 @@
+from lego.utils.managers import BasisModelManager
+
+
+class LendingInstanceManager(BasisModelManager):
+    def create(self, *args, **kwargs):
+        from lego.apps.lending.notifications import LendingInstanceNotification
+        from lego.apps.users.models import Membership, User
```

Comment:
Idk how python works, is it really a good idea to import these inside the function?

Reply:
It's like a lazy import to avoid circular dependencies, I think it's done other places too

Reply:
Ah, that makes sense:)